src/Sylius/Bundle/ResourceBundle/Controller/RequestConfigurationFactory.php

```diff
@@ -74,12 +74,23 @@ private function parseApiParameters(Request $request)
     {
         $parameters = [];
 
-        if (preg_match(self::API_VERSION_REGEXP, $request->headers->get(self::API_VERSION_HEADER), $matches)) {
-            $parameters['serialization_version'] = $matches['version'];
+        $apiVersionHeaders = $request->headers->get(self::API_VERSION_HEADER, null, false);
+        foreach ($apiVersionHeaders as $apiVersionHeader) {
+            if (preg_match(self::API_VERSION_REGEXP, $apiVersionHeader, $matches)) {
+                $parameters['serialization_version'] = $matches['version'];
+            }
         }
 
-        if (preg_match(self::API_GROUPS_REGEXP, $request->headers->get(self::API_GROUPS_HEADER), $matches)) {
-            $parameters['serialization_groups'] = array_map('trim', explode(',', $matches['groups']));
+        $serializationGroups = [];
+        $apiGroupsHeaders = $request->headers->get(self::API_GROUPS_HEADER, null, false);
+        foreach ($apiGroupsHeaders as $apiGroupsHeader) {
+            if (preg_match(self::API_GROUPS_REGEXP, $apiGroupsHeader, $matches)) {
+                $serializationGroups = array_merge($serializationGroups, array_map('trim', explode(',', $matches['groups'])));
+            }
+        }
+
+        if (count($serializationGroups)) {
```

Comment:
Is it needed? `array_merge` will return an empty array in the worst case. I think we should remove this if.